src/apiBase/APIResponse.es6.js

```diff
@@ -85,7 +85,7 @@ export class APIResponseBase {
   appendResponse() { throw new Error('Not implemented in base class'); }
 }
 
-export default class APIResponse extends APIResponseBase {
+export class APIResponse extends APIResponseBase {
```

Comment:
I figured APIResponse was no more important than the other classes in this file so I made it a non-default. I can flip this if we disagree.